src/sync/synchronizer.rs

```diff
@@ -903,7 +903,7 @@ fn apply_deletes_to_localmap(to_delete: &mut Update, table_synchronizer: &mut Sy
         let in_mem_inner_map = table_synchronizer
             .in_memory_map
             .entry(delete.outer_key.clone())
-            .or_insert_with(HashMap::new);
+            .or_default();
```

Comment:
Updated this line as per the suggestion provided in the failed build log.
![image](https://github.com/pravega/pravega-client-rust/assets/122247323/e8fef243-6dd8-45d1-921e-68d58d90ce5c)